db/migrations/20220609044825-move-github-app-id-to-subscriptions.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		await queryInterface.removeColumn("Installations", "githubAppId");
```

Comment:
I assume this column is for GHE at the moment and therefore NO production data in it, right? Otherwise we can preserve this column until prod data is migrated.

Reply:
Yep new column added specifically for ghe so no prod data.